.github/workflows/build-test.yml

```diff
@@ -27,7 +27,7 @@ on:
   workflow_dispatch:
 env:
   NODE_VERSION: 14.16.0
-  PYTHON_VERSION: 3.8
+  PYTHON_VERSION: 3.9
```

Comment:
I'm also going to look at adding vNext (3.10) in another change. I don't think they both need full coverage though, so might just add a subset of tests in that.

Reply:
Whoops, this is for vsix build. Updating the other spot.